pkg/command/repair/cmd.go

```diff
@@ -65,7 +65,14 @@ func newCommand(client *managerclient.Client, update bool) *command {
 	}
 	cmd.init()
 	cmd.RunE = func(_ *cobra.Command, args []string) error {
-		return cmd.run(args)
+		err := cmd.run(args)
+		if errors.Is(err, managerclient.ErrClientError) {
+			fmt.Fprintf(cmd.OutOrStderr(), "One of the nodes is down. Task cannot be created/updated.\n"+
+				"Please use --ignore-down-hosts flag (https://manager.docs.scylladb.com/stable/sctool/repair.html#ignore-down-hosts)"+
+				" if you want to skip the repair on this node.\n\n")
+		}
+
+		return err
```

Comment:
Is this link needed here? If user want to get this information, it can be obtained with `--help`. Not to say that the information is `Do not repair nodes that are down i.e. in status DN.`.

Also, I think that ifing out errors both on server and cli tool side is a double work. Can't just return a descriptive error directly from server?

Reply:
There is a tiny problem in returning the descriptive error from server directly.
The normal approach is just to chain errors and return them the way
`error1 : error2 : .... etc` 

It's not the most humane readable way of informing end-user about the root cause of the problem.

To make it human-readable, the error must be parsed and print out in bit better organized way.

Reply:
I will change it bit different way.
Create the general error = "host is down". Put it in chain and look for this error on parsing the output on Sctool repair cmd.